shopify/limits.py

```diff
@@ -0,0 +1,39 @@
+import shopify
+
+class Limits(object):
+    CREDIT_LIMIT_HEADER_PARAM = 'x-shopify-shop-api-call-limit'
+
+    @classmethod
+    def credit_left(cls):
+        return cls.credit_limit() - cls.credit_used()
+
+    available_calls = credit_left
+
+    @classmethod
+    def is_credit_maxed(cls):
+        return cls.credit_left() <= 0
+
+    is_maxed = is_credit_maxed
```

Comment:
You could make this a read-only property instead of a function like in `base.py`, in which case you can use [this syntax](https://github.com/Shopify/shopify_python_api/blob/3fc6c0187b8cf37f527f56b2a8b8c71eab8d8840/shopify/base.py#L57-L58) here, e.g.
```python
    is_maxed = property(is_credit_maxed, None, None, "Docstring")
```

Reply:
I think I may just remove the aliasing all together. It doesn't feel very "pythonic" to me.